code/__DEFINES/dcs/signals.dm

```diff
@@ -516,6 +516,8 @@
 // /mob/living/simple_animal/hostile signals
 #define COMSIG_HOSTILE_ATTACKINGTARGET "hostile_attackingtarget"
 	#define COMPONENT_HOSTILE_NO_ATTACK (1<<0)
+///from base of mob/living/simple_animal/hostile/regalrat: (mob/living/L)
```

Comment:
Are you sure it's `/mob/living` and not `/mob/living/simple_animal/hostile/regalrat`?